src/cli/index.ts

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env node
 
 import cli from '@gql.tada/cli-utils';
-cli();
+(cli as any).default();
```

Comment:
Are the types erroneously generated now?

Reply:
Confirmed types for `gql-data-cli.d.ts` are the same as they were previous to this change 👍🙂